herbert_core/utilities/general/time_waster.m

```diff
@@ -0,0 +1,41 @@
+function yout = time_waster (yin, nloop)
+% Function to perform a very expensive operation that is almost an identity
+%
+%   >> yout = time_waster (yin)         % Perform the operation once
+%   >> yout = time_waster (yin, nloop)  % perform the operation nloop times
+%
+% The output is an array in which each element differs by a factor of less
+% than or equal to 10^-13 of the original, irrespective of the value of
+% nloop.
+%
+% The time it takes for each loop is approximately 25x that of
+% exponentiation.
+%
+% Input:
+% ------
+%   yin     Array of values
+%   nloop   Number of times to perform the operation
+%
+% Output:
+% -------
+%   yout    Output array
+
+
+if nloop~=0
```

Comment:
Don't strictly need to check here, 
```matlab
for i=1:0
 ...
end
```
Should not execute. It may, however, spend some time doing the elementwise product, I guess.